src/GitHash.hs

```diff
@@ -119,6 +121,11 @@ giCommitMessage = _giCommitMessage
 giDescribe :: GitInfo -> String
 giDescribe = _giDescribe
 
+-- | The output of @git describe --always --tags@ for the most recent commit.
+--
```

Comment:
Please add a `@since` comment, ChangeLog entry and version bump

Reply:
Done

Reply:
Done